integration/spark/src/test/common/java/io/openlineage/spark/agent/lifecycle/SparkReadWriteIntegTest.java

```diff
@@ -430,6 +435,43 @@ public void testWithLogicalRdd(@TempDir Path tmpDir, SparkSession spark)
         .hasFieldOrPropertyWithValue("namespace", "file");
   }
 
+  @Test
+  public void testCreateDataSourceTableAsSelect(@TempDir Path tmpDir, SparkSession spark)
```

Comment:
Is this test related? Or you're just adding some test coverage?

Reply:
It's a separate test specifically related to the fix. It's not the test that executes the pyspark script, if that's your confusion. The `writeTestDataToFile` method we call on line 441 writes data that fits the schema in the assertion.

Reply:
Ah got it 👌